src/components/users/ForgotPwd.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from 'react';
+import { func, object } from 'prop-types';
+import Loader from 'react-loader-spinner';
+import { useDispatch, useSelector } from 'react-redux';
+
+import '../../style/App.scss';
+import './user-form.scss';
+import FormInput from '../common/FormInput';
+import { userRequest, userActions } from '../../actions/user.actions';
+import { userConstants } from '../../constants/user.constants';
+
+const ForgotPwd = ({ intl, setForgotPwd }) => {
+  const dispatch = useDispatch();
+  const [inputs, setInputs] = useState({ email: '' });
+  const { email } = inputs;
+
+  const [success, setSuccess] = useState(false);
+
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  const { loading, emailSent, errorMsg } = useSelector((state) => state.user);
+
+  const showError = isSubmitted && errorMsg;
+
+  useEffect(() => {
+    if (emailSent) {
+      setSuccess(true);
+      setIsSubmitted(false);
+    }
+  }, [emailSent, setSuccess, setIsSubmitted]);
+
+  const handleChange = ({ target: { name, value } }) => {
+    setIsSubmitted(false);
+    setInputs((inputs) => ({ ...inputs, [name]: value }));
+  };
+
+  const handleGoBack = () => {
+    setForgotPwd(false);
+    setSuccess(false);
+    setInputs((inputs) => ({ ...inputs, email: '' }));
+    dispatch({ type: userConstants.USER_REQUEST_SUCCESS });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setIsSubmitted(true);
+    if (email) {
+      dispatch(userRequest());
+      dispatch(userActions.resetPwd(email));
+    }
+  };
+
+  return (
+    <div align="center" className="login-page-container__forgot_pwd">
+      <p className="login-page-container__app-subtitle">
+        {intl.formatMessage({ id: 'userform.forgotpwd.text' })}
+      </p>
+      {!success ? (
```

Comment:
Just for readability is preferred to ask for truthy values when using conditional render.
```suggestion
      {success ? (
      you must invert the order of your components here
      )
```